.babelrc

```diff
@@ -7,6 +7,7 @@
     "react-hot-loader/babel",
     ["babel-plugin-styled-components", {
       "displayName": true
-    }]
+    }],
+    "transform-object-rest-spread"
```

Comment:
`stage-2` covers this and in order to use other specifications as well, I think adding `stage-2` instead of `transform-object-rest-spread` should be better

e.g.
```
"presets": [
    "env",
    "react",
    "stage-2"
  ],
```